datastore-api/tests/conftest.py

```diff
@@ -76,12 +76,12 @@ def conceptnet_kg(kg_name):
     return Datastore(
         name=kg_name,
         fields=[
-            DatastoreField(name="name", type="keyword"),
-            DatastoreField(name="type", type="keyword"),
+            DatastoreField(name="name", type="text"),
+            DatastoreField(name="type", type="text"),
             DatastoreField(name="description", type="text"),
-            DatastoreField(name="weight", type="double"),
-            DatastoreField(name="in_id", type="keyword"),
-            DatastoreField(name="out_id", type="keyword"), 
+            DatastoreField(name="weight", type="float"),
+            DatastoreField(name="in_id", type="text"),
+            DatastoreField(name="out_id", type="text"), 
```

Comment:
Why do we need such changes? And actually these are different from https://github.com/UKP-SQuARE/square-core/blob/d659549f85ca2f769353c6abea9d383ea8261d66/datastore-api/app/core/kgs/connector.py#L57-L67